client/landing/stepper/declarative-flow/internals/steps-repository/videopress-onboarding-intent/videopress-onboarding-intent-modal-portfolio.tsx

```diff
@@ -36,11 +45,32 @@ const VideoPressOnboardingIntentModalPortfolio: React.FC< IntroModalContentProps
 
 		if ( planProductObject ) {
 			// eslint-disable-next-line @wordpress/valid-sprintf
-			getStartedText = sprintf(
-				/* translators: Price displayed on VideoPress intro page. %s is monthly price. */
-				translate( 'Get started - from %s/month' ),
-				planProductObject.price
-			);
+			getStartedText = config.isEnabled( 'videomaker-trial' )
+				? translate( 'Start free trial' )
```

Comment:
```suggestion
				? translate( 'Start a free trial' )
```

Completely up to you, but I think this reads a little more friendly, less programmer-y.